homeassistant/components/tuya/climate.py

```diff
@@ -277,6 +278,10 @@ def __init__(
             if self.find_dpcode(DPCode.SWITCH_VERTICAL, prefer_function=True):
                 self._attr_swing_modes.append(SWING_VERTICAL)
 
+        self._attr_supported_features |= (
+            ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
```

Comment:
Right now, this is a fake feature (partly) because of the old behavior (see `turn_on` / `turn_off` methods).

Instead of just adding these, the integration should be adapted accordingly.

../Frenck

Reply:
Modified also the methods so it's adapted accordingly. Thanks